etc/config/zig.defaults.properties

```diff
@@ -1,9 +1,12 @@
-compilers=/usr/bin/zig
+compilers=zig:/usr/bin/zig
```

Comment:
Maybe I'm misunderstanding but there's no compiler here named zig?

Reply:
or is this trying to find `zig` on the path? (which I don't think works)

Reply:
> or is this trying to find zig on the path? (which I don't think works)

Yes, it picked up my local zig after this change but maybe this is not what I think it is.

Its also what I did for `odin`, https://github.com/compiler-explorer/compiler-explorer/blob/main/etc/config/odin.defaults.properties#L1 (which I might have copied from some other place, not sure)

Reply:
Oh interesting, I was always under the impression we didn't support this, but maybe it does work?

Anyhow since it's for local I think I'm cool with this change.